.github/workflows/ci-lint-test.yml

```
@@ -52,7 +52,7 @@ jobs:
         shell: bash
     strategy:
       matrix:
-        node: [12, 14, 15]
+        node: [12, 14, 15, 16]
```

```suggestion
        node: [12, 14, 16]
```
Node 15 is not an LTS release, and we have our next LTS release now. We need not use Node 15 anymore.